PredixMobileUIKit/PredixTimeSeriesView+IBInspectable.swift

```diff
@@ -21,22 +21,35 @@ extension PredixTimeSeriesView {
     
     // MARK: - private functions
     private func initializeWithDummyData() {
-        var tags: [TimeSeriesTag] = []
+        var tags: [LineChartDataSet] = []
         let range = 8
         let upperRange = 2018
         let lowerRange = upperRange - range
+        var colorCounter: Int = 0
         for i in 1...3 {
-            var dataPoints: [TimeSeriesDataPoint] = []
+            var dataPoints: [ChartDataEntry] = []
             for j in 0...range {
                 let time = Double(lowerRange + j)
                 let measure = Double((arc4random_uniform(UInt32(115)) + UInt32(50)) )
```

Comment:
As prepareForInterfaceBuilder calls initializeWithDummyData, the data set in IB is constantly changing. This can make it difficult to tell what's changing as a user adjusts IB properties. Perhaps this could just be a static list, or only generated once?